pkg/controller/replicaset/replica_set_test.go

```diff
@@ -248,24 +225,6 @@ func TestSyncReplicaSetDoesNothing(t *testing.T) {
 	validateSyncReplicaSet(t, &fakePodControl, 0, 0, 0)
 }
 
-func TestSyncReplicaSetDeletes(t *testing.T) {
```

Comment:
Make sure your integration tests contain a step where you ensure N replicas, and then scale down to <N.

Reply:
sure, i will add the step.

Reply:
This step should have been added by: 
```
scaleRS(t, c, rs, 3)
scaleRS(t, c, rs, 0)
scaleRS(t, c, rs, 2)
```